src/goCover.ts

```diff
@@ -140,6 +148,13 @@ export function getCoverage(coverProfilePath: string, showErrOutput: boolean = f
 				let fileRange = data.match(/([^:]+)\:([\d]+)\.([\d]+)\,([\d]+)\.([\d]+)\s([\d]+)\s([\d]+)/);
 				if (!fileRange) return;
 
+				// I really don't know how to fix this elsewhere. I get an underscores in front o thef path (on macOS)
+				//  in go-code-cover file. This fixes it.
+				// _/Users/me/.../file.go => /Users/me/.../file.go
+				if (fileRange[1].charAt(0) === '_') {
```

Comment:
I don't have this problem in my Mac.

I did see `_` in test output once long ago when I had opened the GOPATH in VS Code instead of just my Go project. Is that the same case with you?